forms/FormField.php

```diff
@@ -255,6 +251,22 @@ class FormField extends RequestHandler {
 	 */
 	protected $schemaData = [];
 
+	private static $casting = array(
+		'FieldHolder' => 'HTMLFragment',
+		'Field' => 'HTMLFragment',
```

Comment:
If Field is already using `renderTemplate()`, which returns an HTMLFragment, why does it casting? Is it just here for the benefit of subclasses that don't use a template?